### PR TITLE
Added some setter functions.

### PR DESCRIPTION
**Pull request by Shubhayu-Das**
#Features added

I have added the following functions:
`setDrawMode(mode)`: set the drawing mode of canvas to fill/stroke. Default value is stroke. An error in the value of mode results in choosing "stroke" by default.

`draw()`: which draws the shapes according to the `drawMode` variable.

`setLineThickness(width = 1)`: changes the width of the stroke line while in stroke mode. Default value is 1.

`setColor(color)`: changes the color of the canvas using `canvas.fillStyle()` and `canvas.strokeStyle()`.
Default value is black, if a wrong color is passed.

### CODE REVIEW

File: canvas.js [hunk 1]
```diff
@@ -19,8 +19,14 @@
         canvas.mouseDown // Boolean to check if mouse is pressed down
         canvas.mouseDownX // Last Position of mouse press x coordinate
         canvas.mouseDownY // Last Position of mouse press y coordinate
+        canvas.drawMode  // Fill in a shape or just draw the border
 
     Helpful Canvas Functions
+        canvas.setDrawMode(mode)   // Set the drawMode variable to "stroke" or "fill"
+        canvas.setColor(color)  // Set the color of the shape(s) to the given color. Default color is black(#000000)
```
Comment (satu0king) [inline canvas.js:27]: Does colors have to be in hex only or can it also be like "red" and  "rgba(0.3,0.3,0.3,0.8)" ? 
If so add the different forms of defining colors in the documentation

**Review by satu0king — CHANGES_REQUESTED** [review]
Looks good, just look into the one thing.

File: canvas.js (hunk 1 above)
Comment (Shubhayu-Das) [reply canvas.js:27]: It can be any string. I tested it with rgba as well. It works on any valid CSS color code type.

Comment (satu0king) [reply canvas.js:27]: Add it to the documentation then.

Comment (Shubhayu-Das) [reply canvas.js:27]: Anything else?

Comment (satu0king) [reply canvas.js:27]: No, just color documentation update

**Review by satu0king — CHANGES_REQUESTED** [review]

**Comment by satu0king** [comment]
awesome

**Pull request merged**